lib/mini_profiler/timer_struct/client.rb

```diff
@@ -16,10 +17,7 @@ def self.init_instrumentation
         # used by Railtie to instrument asset_tag for JS / CSS
         def self.instrument(name, orig)
           probe = "<script>mPt.probe('#{name}')</script>"
-          wrapped = probe
-          wrapped << orig
-          wrapped << probe
-          wrapped
+          probe + orig + probe + orig
```

Comment:
Warning here, there is an extra `orig`. It should be `probe + orig + probe`.

Reply:
check the test here https://github.com/MiniProfiler/rack-mini-profiler/blob/master/spec/lib/timer_struct/client_timer_struct_spec.rb#L41

Reply:
Indeed. Sorry for the burden.